src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs

```diff
@@ -312,6 +312,21 @@ private IntPtr GetParentWindow(CoreUIParent uiParent)
                         var errorMessage = "Could not acquire token silently.";
                         msalTokenResponse = WamAdapters.HandleResponse(result, authenticationRequestParameters, _logger, errorMessage);
                     }
+
+                    if (acquireTokenSilentParameters.ForceRefresh && !string.IsNullOrEmpty(msalTokenResponse.AccessToken))
+                    {
+                        authParams.AccessTokenToRenew = msalTokenResponse.AccessToken;
+
+                        using (NativeInterop.AuthResult result = await s_lazyCore.Value.AcquireTokenSilentlyAsync(
```

Comment:
So this is calling `AcquireTokenSilentlyAsync` on MSAL Runtime a second time but we are now setting the `AccessTokenToRenew` property. Are there plans to enable MSAL runtime to respond to the `ForceRefresh` value the first time so we dont need to make 2 calls?

Reply:
The plan is to expose AccessTokenToRenew() to reduce the chance of WithForceRefresh() being abused.